Add note about handling multiple methods in solid-start API handler (#724)
solidjs/solid-docs-next: src/routes/solid-start/building-your-application/api-routes.mdx

- `params`: Object that contains the dynamic route parameters. For example, if the route is `/api/users/:id`, and the request is made to `/api/users/123`, then `params` will be `{ id: 123 }`.
- `fetch`: An internal `fetch` function that can be used to make requests to other API routes without worrying about the `origin` of the URL.

> An API route is expected to return JSON or a `Response` object. To handle all methods, you should define a handler function and bind multiple methods to it, like so:

Reviewer: ```suggestion
An API route is expected to return JSON or a `Response` object. 
In order to handle all methods, you can define a handler function that binds multiple methods to it:
```
